docs/docs/workflows/troubleshooting.md

```diff
@@ -95,49 +95,14 @@ check if the encountered [issue is known](https://github.com/edgelesssys/constel
 
 ## Diagnosing issues
 
-### Cloud logging
+### Logs
 
-To provide information during early stages of a node's boot process, Constellation logs messages to the log systems of the cloud providers. Since these offerings **aren't** confidential, only generic information without any sensitive values is stored. This provides administrators with a high-level understanding of the current state of a node.
+To get started on diagnosing issues with Constellation, it's often helpful to collect logs from nodes, pods, or other resources in the cluster. Most logs are available through Kubernetes' standard
+[logging interfaces](https://kubernetes.io/docs/concepts/cluster-administration/logging/).
 
-You can view this information in the following places:
+To debug issues occurring at boot time of the nodes, you can use the serial console interface of the CSP.
```

Comment:
I think this option is actually disabled everywhere where its possible for non debug deployments.

Reply:
Also seeing the output? Or only logging in?

Reply:
Verifying this currently. I think we should have a read-only console on all CSPs

Reply:
Ah, you mean the serial console is disabled via the CSP, but not in a CC-secure way, right?

Reply:
Well, the read-only configuration is embedded in our image and thus also "CC-secure", but the cloud provider may preserve the logs after boot, which we seemingly explicitly turn off where possible. However, the serial console in all cases is read-only and, using our image, doesn't disclose any sensible information, so I don't think it's a problem.

Reply:
Yes, I didn't mean that there's a security issue. I only remembered that, in addition, we may have turned off serial console access via VM settings as a kind of best-effort approach.